action.yml

```diff
@@ -22,11 +22,10 @@ inputs:
 runs:
   using: composite
   steps:
-    - name: Run gcov2lcov
+    - shell: bash
       run: ${{ github.action_path }}/entrypoint.sh
-      shell: bash
+      working-directory: ${{ inputs.workspace }}
```

Comment:
* please document `workspace` input in the `README`
* what happens if the `workspace` input is NOT set (it is not required)?

Reply:
the repo use case (test data in subdirectory)

maybe need monorepo this feature (change working directory)

Reply:
in fact, we are all is optional inputs.

see https://github.com/NiceLabs/go-smpp/blob/fbcbef8b5dcac336a33783408707f56c653199fd/.github/workflows/build.yml#L28-L29

Reply:
> * what happens if the `workspace` input is NOT set (it is not required)?

default case working directory is `$GITHUB_WORKSPACE`